### PR TITLE
fix(topology): fix to layout nodes correctly on first load

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
### Description
Initial loading of the topology view can, in instances, fail to layout the nodes centered on the view (in particular, in the [Janus Showcase application](https://github.com/janus-idp/backstage-showcase.git ).

### Fix
The fix is to wait until the graph surface area has been initialized and sized so that the graph component has dimensions. This allows the layout of the nodes to be set based on the current graph size and centered on the graph surface.

/cc @invincibleJai

### CODE REVIEW

**Comment by invincibleJai** [comment]
Thanks a lot @jeff-phillips-18 👍 

there are some lint failures https://github.com/janus-idp/backstage-plugins/actions/runs/7847970308/job/21418079182?pr=1194#step:3:429

File: plugins/topology/src/components/Topology/TopologyViewWorkloadComponent.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {
   useVisualizationController,
   VisualizationSurface,
 } from '@patternfly/react-topology';
+import { observer } from 'mobx-react';
```
Comment (invincibleJai) [inline plugins/topology/src/components/Topology/TopologyViewWorkloadComponent.tsx:14]: can we use the one from '@patternfly/react-topology' as this also exposes `observer`

File: plugins/topology/src/components/Topology/TopologyViewWorkloadComponent.tsx [hunk 2]
```diff
@@ -51,18 +55,21 @@ const TopologyViewWorkloadComponent = ({
...
-      controller.fromModel(model, true);
+    const model = {
+      graph: {
+        id: 'g1',
+        type: 'graph',
+        layout,
+      },
+    };
+    controller.fromModel(model, false);
+  }, []);
```
Comment (invincibleJai) [inline plugins/topology/src/components/Topology/TopologyViewWorkloadComponent.tsx:66]:
```suggestion
    controller.fromModel(model, false);
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```

**Review by invincibleJai — APPROVED** [review]
/lgtm

**Review by gashcrumb — APPROVED** [review]

**Review by gashcrumb — APPROVED** [review]

**Pull request merged**